vendor/github.com/longhorn/backupstore/deltablock.go

```diff
@@ -392,6 +392,13 @@ func RestoreDeltaBlockBackup(config *DeltaRestoreConfig) error {
 		return fmt.Errorf("Read invalid volume size %v", vol.Size)
 	}
 
+	if _, err := os.Stat(volDevName); err == nil {
+		logrus.Warnf("File %s for the restore exists, will remove and re-create it", volDevName)
+		if err := os.Remove(volDevName); err != nil {
+			return fmt.Errorf("failed to clean up the existing file %v before restore: %v", volDevName, err)
+		}
+	}
+
```

Comment:
This modification is not from me, it was from [this commit](https://github.com/longhorn/backupstore/commit/31a628633b8f0c6ec1abac929d6763d36b5ec1c4)